e2e/app/eoa/static.go

```diff
@@ -0,0 +1,176 @@
+			Address:       devnetCreate3Deployer,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(10),
+			TargetBalance: big.NewInt(100)},
+
+		{
+			Type:          TypeRemote,
+			Role:          RoleDeployer,
+			Address:       devnetDeployer,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(10),
+			TargetBalance: big.NewInt(100),
+		},
+		{
+			Type:          TypeRemote,
+			Role:          RoleAdmin,
+			Address:       devnetAdmin,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(10),
+			TargetBalance: big.NewInt(100),
+		},
+		{
+			Type:          TypeWellKnown,
+			Role:          RoleRelayer,
+			Address:       crypto.PubkeyToAddress((anvil.DevPrivateKey5()).PublicKey),
+			PrivateKey:    anvil.DevPrivateKey5(),
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(10),
+			TargetBalance: big.NewInt(100),
+		},
+		{
+			Type:          TypeWellKnown,
+			Role:          RoleMonitor,
+			Address:       crypto.PubkeyToAddress((anvil.DevPrivateKey6()).PublicKey),
+			PrivateKey:    anvil.DevPrivateKey6(),
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(10),
+			TargetBalance: big.NewInt(100),
+		},
+		{
+			Type:          TypeRemote,
+			Role:          RoleFbDev,
+			Address:       addr(fbDev),
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(10),
+			TargetBalance: big.NewInt(100),
+		},
+	},
+	netconf.Staging: {
+		{
+			Type:          TypeRemote,
+			Role:          RoleCreate3Deployer,
+			Address:       stagingCreate3Deployer,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(2),
+			TargetBalance: big.NewInt(10)},
+
+		{
+			Type:          TypeRemote,
+			Role:          RoleDeployer,
+			Address:       stagingDeployer,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(2),
+			TargetBalance: big.NewInt(10),
+		},
+		{
+			Type:          TypeRemote,
+			Role:          RoleAdmin,
+			Address:       stagingAdmin,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(1),
+			TargetBalance: big.NewInt(5),
+		},
+		{
+			Type:          TypeSecret,
+			Role:          RoleRelayer,
+			Address:       stagingRelayer,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(1),
+			TargetBalance: big.NewInt(5),
+		},
+		{
+			Type:          TypeSecret,
+			Role:          RoleMonitor,
+			Address:       stagingMonitor,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(1),
+			TargetBalance: big.NewInt(5),
+		},
+		{
+			Type:          TypeRemote,
+			Role:          RoleFbDev,
+			Address:       addr(fbDev),
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(5),
+			TargetBalance: big.NewInt(10),
+		},
+	},
+	netconf.Testnet: {
+		{
+			Type:          TypeRemote,
+			Role:          RoleCreate3Deployer,
+			Address:       testnetCreate3Deployer,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(2),
+			TargetBalance: big.NewInt(10),
```

Comment:
I don't think create3 deployers need this much - they only ever do one deployment